.github/workflows/ci.yml

```diff
@@ -152,8 +147,38 @@ jobs:
 
             -   name: Run Evaluator for all branches
                 run: vendor/bin/evaluator evaluate --exclude-checkers=SPRYKER_DEV_PACKAGES_CHECKER --format=compact
+    js-validation:
+        if: >
+            contains(github.event.pull_request.labels.*.name, 'run-npm-ci')
+            || contains(github.event.pull_request.labels.*.name, 'run-latest-ci')
+        name: "NPM validation"
+        runs-on: ubuntu-22.04
+        strategy:
+            fail-fast: false
+
+        env:
+            APPLICATION_ENV: ci.mysql
+            APPLICATION_STORE: DE
```

Comment:
Do we need a store and a region together? I assume store is not needed if DMS is ON and visa versa for the region